packages/authentication/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@loopback/context": "^4.0.0-alpha.24",
     "@loopback/core": "^4.0.0-alpha.26",
+    "@loopback/openapi-v2": "1.0.0-alpha1",
```

Comment:
@jannyHou The version number should have been `1.0.0-alpha.1` (notice the dot between "alpha" and "1"), this is the scheme we are using in other packages (look at lines 26 and 27 above for examples).

Reply:
I think Raymond already created a PR to fix the version.